### PR TITLE
[coreboot chapter] updated build instructions

### LINKED ISSUE #35: Build error. Says "not enough arguments"
Hi Team,
I just started on u-root with a focus on Linuxboot. I am not familiar with Go language. I am also new to Linux. At this point everything is pretty much "learn as you go" for me :-) I am following instructions from https://github.com/linuxboot/book/blob/master/coreboot.u-root.systemboot/README.md

When I reach the command "u-root -build=bb core github.com/systemboot/systemboot/{uinit,localboot,netboot}" it fails with an error "not enough arguments". See below the full terminal output. Any pointers please?

madhan@madhan-HP-Notebook:~/work$ u-root -build=bb core github.com/systemboot/systemboot/{uinit,localboot,netboot}
2019/11/19 17:08:52 Disabling CGO for u-root...
2019/11/19 17:08:52 Build environment: GOARCH=amd64 GOOS=linux GOROOT=/usr/local/go GOPATH=/home/madhan/work CGO_ENABLED=0
2019/11/19 17:08:54 Filename is /tmp/initramfs.linux_amd64.cpio
2019/11/19 17:09:53 error building: error building go package in "/home/madhan/work/src/github.com/u-root/u-root/bb": # github.com/systemboot/systemboot/pkg/bootconfig
../../../systemboot/systemboot/pkg/bootconfig/bootconfig.go:93:27: not enough arguments in call to multiboot.Load
	have (bool, string, string, []string)
	want (bool, string, string, []string, *ibft.IBFT)
, exit status 2

Thanks in advance,
Madhan

### PR DESCRIPTION
**Pull request by insomniacslk**
Removed references to systemboot that is now merged in u-root.

fixes #35 

Signed-off-by: Andrea Barberio <insomniac@slackware.it>

### CODE REVIEW

**Comment by aroobam** [comment]
Hi insomniacslk, Thanks, that works. I am past that part. initramfs is generated. I will try further steps and update tomorrow. It is almost midnight for me here.
Your updates need a little tweak. The line "u-root -build=bb core cmds/boot/{uinit,localboot,fbnetboot}" should be "u-root -build=bb core github.com/u-root/u-root/cmds/boot/{uinit,localboot,fbnetboot}", i.e. full path for it to work as per the flow of the previous steps and working directory.

**Comment by insomniacslk** [comment]
Ok - that worked for me without full path, but probably just because I was already in the u-root source directory. I'll update and merge. Thanks for the feedback!

**Comment by insomniacslk** [comment]
I'll wait for review for 24 hours and then merge

File: coreboot.u-root.systemboot/README.md [hunk 1]
```diff
@@ -26,33 +26,32 @@ The final image is built on top of multiple open-source components:
 * [coreboot](https://coreboot.org), used for the platform initialization. Silicon and DRAM initialization are done here.
 * [Linux](https://kernel.org), used to initialize peripherals and various device drivers like file systems, storage and network devices; network stack; a multiuser and multitasking environment.
 * [u-root](https://u-root.tk), an user-space environment that provides basic libraries and utilities to work in a Linux environment.
-* [systemboot](https://systemboot.org), an additional set of libraries and tools on top of u-root, that provide a bootloader behaviour for various booting scenarios.
+* ~~[systemboot](https://systemboot.org), an additional set of libraries and tools on top of u-root, that provide a bootloader behaviour for various booting scenarios.~~ systemboot was merged in u-root.
```
Comment (orangecms) [inline coreboot.u-root.systemboot/README.md]: "merged into" instead of "merged in" :)

Comment (insomniacslk) [reply coreboot.u-root.systemboot/README.md]: done - thanks!

**Pull request merged**